core/src/main/java/org/mobilitydata/gtfsvalidator/notice/NonAsciiOrNonPrintableCharNotice.java

```diff
@@ -18,7 +18,7 @@
 
 import com.google.common.collect.ImmutableMap;
 
-/** A row in the input file has a different number of values than specified by the CSV header. */
+/** A value of filed with type `id` contains non ASCII or non printable characters. */
```

Comment:
I think there's a typo in "filed". Do you meant field?

Reply:
Indeed! thanks